lib/widgets/molecules/profile_experience_and_education.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+
+class ExperienceAndEducation extends StatelessWidget {
+  const ExperienceAndEducation({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Padding(
+      padding: EdgeInsets.all(Insets.paddingMedium),
+      child: Column(
+        children: [
+          Row(children: [_Experience()]),
+          Row(children: [_Education()]),
+        ],
+      ),
+    );
+  }
+}
+
+class _Experience extends StatelessWidget {
+  static final List<_ExperienceItem> _exampleExperience = [
+    _ExperienceItem(
+      position: "Marketing director",
+      companyName: "SVK Group",
+      start: DateTime.utc(2021, 12),
+      location: "Cleveland, Ohio, USA",
+    ),
+    _ExperienceItem(
+      position: "Senior Marketing Manager",
+      companyName: "SVK Group",
+      start: DateTime.utc(2019, 6),
+      end: DateTime.utc(2021, 11),
+      location: "Cleveland, Ohio, USA",
+    ),
+    _ExperienceItem(
+      position: "Fonder",
+      companyName: "MC Consulting",
+      start: DateTime.utc(2016, 3),
+      end: DateTime.utc(2019, 5),
+      location: "Cleveland, Ohio, USA",
+    ),
+  ];
+
+  const _Experience();
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final items = _exampleExperience
+        .map((exp) => [
+              Text(exp.position, style: theme.textTheme.labelMedium),
+              Text(exp.companyName),
+              Text("(${exp.dateRange}) · ${exp.timeRange}"),
+              Text(exp.location),
```

Comment:
In this entire widget, everything that is not a titleMedium ('My experience' and 'My education') uses textTheme.bodySmall.

The body text that is bold uses fontWeight 700, like this:

```
Text(
    exp.position,
    style: theme.textTheme.bodySmall?.copyWith(
       fontWeight: FontWeight.w700,
    ),
),
```

Reply:
Is... this a problem?

Reply:
Each typescale is optimized for a different use case, body works better for longer lines of text. Also this is what UX used in the design, and we agreed to follow it as closely as possible. It is not a problem per-se and the change is barely noticeable, I am just letting you know that you can get these styles from the mock very easily instead of trying to approximate them.

Reply:
Also, note that the text using the label text style is notably smaller than the rest of the text in the body. If you use the same text style (body) for all of it and then increase the fontWeight for the bold part it will look more consistent.

Sorry if these seem like nits, but this is literally what UI is all about.

Reply:
Changed. I used `FontWeight.bold` as it's the same as `.w700`.